gradio/utils.py

```diff
@@ -225,13 +224,16 @@ def same_children_recursive(children1, chidren2, mapping):
     same_children_recursive(children1, children2, mapping)
 
     for d1, d2 in zip(config1["dependencies"], config2["dependencies"]):
-        for t1, t2 in zip(d1["targets"], d2["targets"]):
+        for t1, t2 in zip(d1.pop("targets"), d2.pop("targets")):
             assert mapping[t1] == t2, "{} does not match {}".format(d1, d2)
```

Comment:
Shall we use f"" strings over here and below?

Reply:
Sg will update